config/wide_responses.toml

```diff
@@ -61,7 +61,7 @@ old_ref = "nan"
 [200]
 Description = "Business type: Civil or Defence"
 Deduced_Data_Type = "str"
-Nullable = "Yes"
+Nullable = "No"
```

Comment:
can you check? 200 and 201 can be null, can't they?

Reply:
Sure - @zorge69 are you able to confirm, or shall I go to BAU?  200 and 201 are Nullable = "No" in DAP

Reply:
I think we know that 200 and 201 are null for instance  0, for example. Also where there is no response. So I'd say it's "Yes", and we can update in DAP.

Reply:
Great, that's been updated in both